tox.ini

```diff
@@ -22,18 +22,22 @@ commands =
     pip install -e .[test]
     pytest --cov=hibpcli --cov=tests --cov-report term-missing  --cov-report html
 
-[testenv:flake8]
-deps =
-    flake8
-    flake8-click
-    flake8-bugbear
-commands =
-    flake8 src tests setup.py
+[testenv:flake8]	
+deps =	
+    flake8	
+    flake8-click	
+    flake8-bugbear	
+commands =	
+    flake8 src tests setup.py	
```

Comment:
We do not need the `flake8` env any more.

But please do not forget to add both `flake8-click` and `flake8-bugbear` as additional dependencies to the flake8 configuration block in the pre-commit-config.yaml file.

You can compare with https://github.com/jugmac00/flask-reuploaded/blob/master/.pre-commit-config.yaml

Reply:
ok so remve from the .ini and add the deps to .yaml

Reply:
remove this?
```yaml
[testenv:flake8]	
deps =	
    flake8	
    flake8-click	
    flake8-bugbear	
commands =	
    flake8 src tests setup.py	

[flake8]	
max-line-length = 88	
ignore = E231  # clashes with black
```

Reply:
You should remove the block with [testenv:flake8] as this configures the env which is now run via pre-commit.

But you should not remove the [flake8] block, as this is just a general configuration for flake8, not matter which tool runs flake8.

You can put configuration in several files in Python, e.g. in a .flake8 or a tox.ini or a setup.cfg or nowadays also in pyproject.toml.

I just keep all my configuration in tox.ini - thats what I am used to and thats what I prefer for my projects.